crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -1,4 +1,5 @@
 {% load static %}
+{% load i18n %}
 {% load with_input_error %}
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
We're not implementing the translation here but flagging since we'll need to update this in a new issue.

Reply:
Maybe I am misunderstanding, but I think this is the load step for {% trans %} tag we use later in the file.